experimental/dds/tree2/src/test/feature-libraries/chunked-forest/chunkedForest.spec.ts

```diff
@@ -86,6 +100,38 @@ describe("ChunkedForest", () => {
 				factory: (schema) => buildChunkedForest(chunker(schema)),
 				skipCursorErrorCheck: true,
 			});
+			describe("moveCursorToPath", () => {
```

Comment:
these tests should be added to the forest test suite (which is run just above this) that is applied to all implementations of forest, not independently implemented for each forest. If there is something exter that a specific forest profides, it can have a dedicated test for that, but as long as something is part of the forest interface, the tests should be in the forest test suite to avoid duplication.

Reply:
This applies to getCursorAboveDetachedFields as well.

Reply:
moved common `moveCursorToPath` test to the forest test suite

Reply:
I've kept the getCursorAboveDetachedFields tests away from the forestTestSuite as the chunkedForest implementation steps into the first node of the forest roots, whereas the objectForest implementation just gets the cursor from the forest root (mapTree). Would there be a better way to test for both cases at the same time?

Reply:
These methods are both implementing the same API. You should write tests that they do what the API says they do. The fact that one of them implements that contract using chunked cursors and the other using map cursors should not matter here: they are supposed to both behave according to the API documentation, and if they don't do that, the test should fail and the implementation should be changed.

Reply:
Ahh yes, that makes sense. I realized I can just use mapCursorField and convert them into jsonableTrees for comparison. I  have moved the tests to the forestSuite.ts.